pom.xml

```diff
@@ -68,7 +68,11 @@
         <junit.version>5.9.2</junit.version>
         <elasticsearch.version>7.17.10</elasticsearch.version>
         <maven-failsafe-plugin.version>2.22.2</maven-failsafe-plugin.version>
-        <maven-resources-plugin.version>3.3.1</maven-resources-plugin.version>
+
+        <!-- maven-resources-plugin version > 3.2.0 introduces changes that
```

Comment:
@matthias-ronge If 3.2.0 introduces those breaking changes, why are you changing to exactly that version here? I suspect either the comment or the version itself in line 74 is wrong. Perhaps you meant version 3.*3*.0 that breaks the functionality?

Reply:
versions **greater** 3.2.0—I will replace the `>` symbol.